.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,6 +3,9 @@
 - [ ] commit messages are meaningful and follow good commit message guidelines
 - [ ] README and other documentation has been updated / added (if needed)
 - [ ] tests have been updated / new tests has been added (if needed)
+- [ ] Screenshots are attached for UI change (if needed)
+- [ ] Functional, Smoke and API tests have been run locally
+- [ ] Full gradle build and run tests with coverage have been completed (if needed)
 
```

Comment:
can we also put evidence of manual testing with screenshot added to jira ticket ?